Chinese_Remainder_Theorem/chinese_remainder_theorem.c

```diff
@@ -0,0 +1,64 @@
+#include <stdio.h>
+
+//function to find minimum number such that num%divisor[i] gives corresponding remainder[i] 
+int minresult( int divisor[], int rem[], int size)
+{
+    int m = 1, M[size], inv[size], i, k, count, x = 0;
+    for(i = 0; i < size; i++)
+    {
+        m = m * divisor[i];
+    }
+    for(i = 0; i < size; i++)
+    {
+        M[i] = m / divisor[i];
+    }
+    for(i = 0; i < size; i++)
+    {   
+        count = 0;
+        k = 1;
+        do
+        {
+            if(((M[i] % divisor[i]) *  k) % divisor[i] == 1)
+            {
+                inv[i] = k;
+                count++;
+            } 
+            k++; 
+        }while(count == 0);  
+    }
+    for(i = 0; i < size; i++)
```

Comment:
please add space after for across the file

Reply:
Sorry. I have made the requested changes but I am unable to squash the commits. So I am making a new PR.

Reply:
Please review PR #2832